www/src/models/statistics.ts

```diff
@@ -1,62 +1,249 @@
-export default class StatisticsModel {
-    static chartColors = {
-      red: 'rgb(250, 20, 47)',
-      orange: 'rgb(203, 75, 22)',
-      yellow: 'rgb(181, 137, 0)',
-      green: 'rgb(133, 153, 0)',
-      blue: 'rgb(38, 139, 210)',
-      magenta: 'rgb(211, 54, 130)',
-      grey: 'rgb(201, 203, 207)',
-      violet: 'rgb(108, 113, 196)',
-      cyan: 'rgb(42, 161, 152)',
+import moment from 'moment';
+import Network from '../services/network';
+
+const colors = {
+  red: 'rgb(250, 50, 47)',
+  orange: 'rgb(203, 75, 22)',
+  yellow: 'rgb(181, 137, 0)',
+  green: 'rgb(133, 153, 0)',
+  blue: 'rgb(38, 139, 210)',
+  magenta: 'rgb(211, 54, 130)',
+  grey: 'rgb(131, 148, 150)',
```

Comment:
i assume we're deliberately deviating from the solarized theme, or is this another one of the 10-ish grey defined colors in the theme?

Reply:
I think it's one of those other grey colors, but I took it directly from the Solar theme, it seems like it deviates from the original Solarized.